notebooks/01_collect_data/raw_to_text.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>Can you add some additional documentation a user understands what each of these update regex are attempting to match?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>